setup.py

```diff
@@ -1,8 +1,17 @@
-# This is only used for readthe docs. Packaging is done with setuptools.
+"""This is only used for readthe docs. Packaging is done with setuptools."""
```

Comment:
This was my fault but packaging is done with maturin. Setup.py did not need to be changed at all since its only there for readthedocs.

Reply:
But it can still be used + `readthedocs` can use `pyproject.toml`